### PR TITLE
Device registration for Play subscription

### LINKED ISSUE #14: Various channels not working
From my testing (using Kodi or the testing framework from #10), the following 12 channels (out of 71 Play live channels) appear not to work:
- **Discovery World** → */play/id/discoverywo*
- **Discovery ID** → */play/id/discoveryid*
- **Discovery Science** → */play/id/discoverysci*
- **History** → */play/id/history*
- **Nat Geo Wild** → */play/id/natgeowild*
- **Extreme Sports** → */play/id/extremesport*
- **Boomerang** → */play/id/boomerang*
- **NickToons** → */play/id/nicktoons*
- **Nick Hits** → */play/id/nickhits*
- **VH1 Music** → */play/id/vh1*
- **MTV Live** → */play/id/mtvnhd*
- **Mezzo** → */play/id/mezzo*

### PR DESCRIPTION
**Pull request by fripsy**
The device will register automatically once a channel  belonging to a valid "Play" subscription is selected. The name of the device will be "YeloPlay".

This fixes #14

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.yelo" name="Yelo" version="1.0.0" provider-name="shycoderX">
```
Comment (michaelarnauts) [inline addon.xml:2]: I'm not sure about this project, but you don't need to change the version. This will be done when a new release is made.

Comment (fripsy) [reply addon.xml:2]: Just put it back to 1.0.0

Comment (dagwieers) [reply addon.xml]: Correct, releases are usually decoupled from features/fixes.

**Comment by dagwieers** [comment]
I am going to merge this, but I would like to have this tested by users that have different subscription levels as well. @shycoderX I assume you are ok with this? If not, I will hear it ;-)

**Pull request merged**